feature_editorial/src/main/java/cm/aptoide/pt/feature_editorial/presentation/EditorialViewScreen.kt

```diff
@@ -0,0 +1,151 @@
+package cm.aptoide.pt.feature_editorial.presentation
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material.Scaffold
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.viewinterop.AndroidView
+import cm.aptoide.pt.aptoide_ui.video.YoutubePlayer
+import cm.aptoide.pt.feature_editorial.R
+import cm.aptoide.pt.feature_editorial.data.ArticleContent
+import cm.aptoide.pt.feature_editorial.data.network.Media
+import cm.aptoide.pt.theme.AptoideTheme
+import coil.compose.rememberImagePainter
+import coil.transform.RoundedCornersTransformation
+
+@SuppressLint("UnusedMaterialScaffoldPaddingParameter")
+@Composable
+fun EditorialViewScreen(viewModel: EditorialViewModel) {
```

Comment:
By the name it could be confused with the EditorialView. I would call the other one EditorialCard (? for example)